lib/main.dart

```diff
@@ -14,7 +14,7 @@ import 'stories/tile_maps/tile_maps.dart';
 import 'stories/utils/utils.dart';
 
 void main() async {
-  final dashbook = Dashbook();
+  final dashbook = Dashbook(theme: ThemeData.dark());
```

Comment:
can we add the title Flame Example here so it doesn't override the one in the HTML?

Reply:
actually let me do that in a followup